capi/Makefile.am

```diff
@@ -1,4 +1,4 @@
-INCLUDES = -I$(top_srcdir) \
+AM_CPPFLAGS = -I$(top_srcdir) \
```

Comment:
https://www.gnu.org/software/automake/manual/html_node/Program-Variables.html

i.e. "Automake already provides some -I options automatically, in a separate variable that is also passed to every compilation that invokes the C preprocessor. In particular it generates ‘-I.’, ‘-I$(srcdir)’, and a -I pointing to the directory holding config.h (if you’ve used AC_CONFIG_HEADERS). "

We use AC_CONFIG_HEADERS, and config.h is in $(top_srcdir), thus we satisfy the last rule.